src/pages/03_FAQ.py

```diff
@@ -33,7 +33,7 @@
 with st.expander("**What is the LaTeX format?**"):
     st.markdown(
     """
-    LaTeX is a document preparation system that is used to render PDFs. ResuLLMe uses LaTeX to render a new AI-curated résumé in a format chosen by you!
+    LaTeX (LAH-tek) is a document preparation system that is used to render PDFs. ResuLLMe uses LaTeX to render a new AI-curated résumé in a format chosen by you!
```

Comment:
I chuckled.  But let’s keep it simple 
```suggestion
    LaTeX is a document preparation system that is used to render PDFs. ResuLLMe uses LaTeX to render a new AI-curated résumé in a format chosen by you!
```